test/framework/mappers/cr_to_cli.go

```diff
@@ -149,7 +148,7 @@ func GetBuildCLIFlags(kogitoBuild *v1alpha1.KogitoBuild) []string {
 		cmd = append(cmd, "--native")
 	}
 
-	image = framework.ConvertImageToImageTag(kogitoBuild.Spec.RuntimeImage)
+	image = kogitoBuild.Spec.RuntimeImage
 	if len(image) > 0 {
 		cmd = append(cmd, "--image-runtime", image)
 	}
```

Comment:
@Kaitou786 Can you please remove this section? Seems to be a copy of line 133-136 (probably a copy paste leftover from some previous adjustment).